modules/installation-osp-verifying-external-network.adoc

```diff
@@ -30,9 +36,10 @@ $ openstack network list --long -c ID -c Name -c "Router Type"
 
 A network with an External router type appears in the network list. If at least one does not, see https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux_OpenStack_Platform/4/html/Installation_and_Configuration_Guide/Configuring_a_Provider_Network1.html[Create an external network].
 
+ifdef::osp-custom,osp-kuryr[]
 [IMPORTANT]
 ====
-If the external network's CIDR range overlaps one of the default network ranges, you must change the matching network range by running the installation program with a custom `install-config.yaml`.
+If the external network's CIDR range overlaps one of the default network ranges, you must change the matching network ranges in `install-config.yaml` before you run the installer.
```

Comment:
s/in install-config.yaml/in the install-config.yaml file
s/installer/installation program